tests/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "resolveJsonModule": true
```

Comment:
why is this needed?

Reply:
To import the package.json I think

Reply:
oh right, I see that in the test file now